### PR TITLE
WIP: Create event for antenna workshop

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: _posts/2017-04-16-radio-antenna-workgroup-meet.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Radio & Antenna Workgroup Meet
+slug: radio-antenna-workgroup-meet
+text: TODO.
```
Comment (dcwalk) [inline _posts/2017-04-16-radio-antenna-workgroup-meet.md]: Maybe just grab the first sentence below?

"A hands on work session to test and configure long-range Ubiquiti antennas."

File: _posts/2017-04-16-radio-antenna-workgroup-meet.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+location: TBA
```
Comment (dcwalk) [inline _posts/2017-04-16-radio-antenna-workgroup-meet.md]: Can we add a location before posting live?

File: _posts/2017-04-16-radio-antenna-workgroup-meet.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+locationLink:
```
Comment (dcwalk) [inline _posts/2017-04-16-radio-antenna-workgroup-meet.md]: OSM!!! ;)

File: _posts/2017-04-16-radio-antenna-workgroup-meet.md (hunk 2 above)
Comment (benhylau) [reply _posts/2017-04-16-radio-antenna-workgroup-meet.md]: Yea this isn't ready, just what we had from the workshop :)

**Comment by dcwalk** [comment]
Hey! This date is passed, @benhylau  is there a status update or am I okay to close?

**Comment by benhylau** [comment]
@dcwalk Please keep this, I will update the date once we have our first hardware meet. Probably around end of month.

**Comment by dcwalk** [comment]
@benhylau -- can we close this PR off for now?

**Comment by dcwalk** [comment]
I'm closing this PR as it was demoing adding events :)

**Pull request closed without merging**